README.md

```diff
@@ -40,141 +40,141 @@ npm run lint-fix # format source files
+| [#126][#126] | [SHOULD define dates properties compliant with RFC 3339][#126] | - | - |
+| [#127][#127] | [SHOULD define time durations and intervals properties conform to ISO 8601][#127] | - | - |
+| [#128][#128] | [SHOULD use standards for country, language and currency codes][#128] | - | - |
+| [#129][#129] | [MUST use lowercase separate words with hyphens for path segments][#129] | - | A |
+| [#130][#130] | [MUST use snake_case (never camelCase) for query parameters][#130] | - | B |
+| [#132][#132] | [SHOULD prefer hyphenated-pascal-case for HTTP header fields][#132] | - | B |
+| [#133][#133] | [MAY use standardized headers][#133] | - | - |
+| [#134][#134] | [MUST pluralize resource names][#134] | - | A |
+| [#135][#135] | [SHOULD not use /api as base path][#135] | - | ? |
+| [#136][#136] | [MUST use normalized paths without empty path segments and trailing slashes][#136] | - | B |
+| [#137][#137] | [MUST stick to conventional query parameters][#137] | - | - |
+| [#138][#138] | [MUST avoid actions — think about resources][#138] | - | A |
+| [#139][#139] | [SHOULD model complete business processes][#139] | - | - |
+| [#140][#140] | [SHOULD define _useful_ resources][#140] | - | - |
+| [#141][#141] | [MUST keep URLs verb-free][#141] | - | A |
+| [#142][#142] | [MUST use domain-specific resource names][#142] | - | - |
+| [#143][#143] | [MUST identify resources and sub-resources via path segments][#143] | - | - |
+| [#144][#144] | [SHOULD only use UUIDs if necessary][#144] | - | - |
+| [#145][#145] | [MAY consider using (non-)nested URLs][#145] | - | - |
+| [#146][#146] | [SHOULD limit number of resource types][#146] | - | B |
+| [#147][#147] | [SHOULD limit number of sub-resource levels][#147] | - | B |
+| [#148][#148] | [MUST use HTTP methods correctly][#148] | - | A |
+| [#149][#149] | [MUST fulfill common method properties][#149] | - | - |
+| [#150][#150] | [MUST use standard HTTP status codes][#150] | - | - |
+| [#151][#151] | [MUST specify success and error responses][#151] | - | B |
+| [#152][#152] | [MUST use code 207 for batch or bulk requests][#152] | - | C |
+| [#153][#153] | [MUST use code 429 with headers for rate limits][#153] | - | C |
+| [#154][#154] | [MUST define collection format of header and query parameters][#154] | - | - |
+| [#155][#155] | [SHOULD reduce bandwidth needs and improve responsiveness][#155] | - | - |
+| [#156][#156] | [SHOULD use `gzip` compression][#156] | - | - |
+| [#157][#157] | [SHOULD support partial responses via filtering][#157] | - | - |
+| [#158][#158] | [SHOULD allow optional embedding of sub-resources][#158] | - | - |
+| [#159][#159] | [MUST support pagination][#159] | - | - |
+| [#160][#160] | [SHOULD prefer cursor-based pagination, avoid offset-based pagination][#160] | - | - |
+| [#161][#161] | [SHOULD use pagination links where applicable][#161] | - | - |
+| [#162][#162] | [MUST use REST maturity level 2][#162] | - | - |
```

Comment:
A too, as verbs and resources should be used correctly?

Reply:
that's a though one to lint :thinking:

Reply:
Yes maybe to big for a start. In addition it includes a lot of interpretation. Let's skip this for now. This should anyway popup in a review ;)